pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -207,7 +207,7 @@ func (c *baseClientImpl) createMultiSearchRequests(searchRequests []*SearchReque
 			header: map[string]interface{}{
 				"search_type":        "query_then_fetch",
 				"ignore_unavailable": true,
-				"index":              c.indices,
+				"index":              strings.Join(c.indices, ","),
```

Comment:
Even though this seems to work and I might have pointed you in a wrong direction (sorry), I am starting to be a bit unsure if that's the right approach. After reading https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html 
```
index
(Optional, string or array of strings) Data streams, indices, and aliases to search. Supports wildcards (*). Specify multiple targets as an array.

If this parameter is not specified, the <target> request path parameter is used as a fallback.
```

We should either use an array here or use the correct `<target>` in the request path parameter. WDYT? We can also discuss with the rest of the squad next week.

Reply:
Definitely! Let's discuss this next week.